benchmarks/make-synchronized.js

```diff
@@ -0,0 +1,11 @@
+import { makeSynchronized } from 'make-synchronized'
+
+/** @type {() => string} */
+const syncFn = makeSynchronized(new URL('native.cjs', import.meta.url))
+
+/** @type {() => string} */
+// More reasonable test?
+// const synchronizeOnFly = filename =>
+//   makeSynchronized('node:fs/promises').readFile(filename, 'utf8')
```

Comment:
Do you think this is more reasonable to test?

Reply:
The API seems awesome.

Reply:
I mean other modules should also test like

```
export default filename => createSyncFn(cjsRequire.resolve('./synckit.worker'))(filename)
```


😄

Reply:
Not really sure how people use.

Reply:
Nope, `createSyncFn` maybe expensive to be called, it should always be preloaded and cached.